docker/Dockerfile

```diff
@@ -2,40 +2,43 @@ ARG cuda_version=9.0
 ARG cudnn_version=7
 FROM nvidia/cuda:${cuda_version}-cudnn${cudnn_version}-devel
 
+# Install system packages
+RUN apt-get update && \
+    apt-get install -y wget git libhdf5-dev g++ graphviz openmpi-bin libgl1-mesa-glx bzip2
+
```

Comment:
Listing the install packages one per line: “sort multine arguments” here (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments)

Reply:
done